scalyr_agent/copying_manager.py

```diff
@@ -821,7 +857,12 @@ def run(self):
 
                             # Now block for the response.
                             blocking_response_time_start = time.time()
-                            (result, bytes_sent, full_response) = get_response()
+                            if self.__config.use_new_ingestion:
+                                result = "success"
+                                bytes_sent = 0
```

Comment:
In theory you should be able to use this value from the IngestionHeadersPb, but this may not be accessible here yet (likely need to make small change to the client so it's available here).